src/python/WMCore/BossAir/Plugins/SimpleCondorPlugin.py

```diff
@@ -289,7 +291,15 @@ def complete(self, jobs):
                             logOutput += '\n\n'
                             for exMsg in listWMExceptionStr(condorFilePath):
                                 logOutput += "\n\n%s\n" % exMsg
-                    condorReport.addError("NoJobReport", 99303, "NoJobReport", logOutput)
+
+                            if condorFile == condorLog:
+                                for condorReason in BasicAlgos.findMagicStr(condorFilePath, '<a n="Reason">'):
+                                    logOutput += condorReason
+                                    if "SYSTEM_PERIODIC_REMOVE" in condorReason or "via condor_rm" in condorReason:
+                                        exitCode = 99400
+                                        exitType = "RemoveByGLIDEIN"
```

Comment:
And here, first comment: how about renaming it to `RemovedByGLIDEIN`?

2nd comment: how about we put an else and set the same exitType but make exitCode = 99401, for instance. Later on we can learn what are the cases for 99401 and apply further filter.